try:
    num=5/0
except:
    print("An error occured")
    raise
# output:
#     num=5/0
# ZeroDivisionError: division by zero
